### PR TITLE
Rotate and Scale

### PR DESCRIPTION
**Pull request by pprogrammer93**
new: rotating and scaling selected object
change: anchor of every new created object is set to be on center,
	it is needed to rotate and scale the object in intuitive manner

### CODE REVIEW

File: src/controller/control.cpp [hunk 1]
```diff
@@ -77,6 +77,18 @@ void readInput(Paint* paint) {
             paint->handleClick();
         }
 
+        if (paint->isObjectSelected()) {
+            if (c == COMMAND_ROTATE_RIGHT) {
+                paint->rotateSelectedObject(0.1);
```
Comment (yonasadiel) [inline src/controller/control.cpp]: jadiin constant di paint.hpp
ROTATION_SPEED = 0.1
SCALING_SPEED = 2.0
ntar rotate -ROTATION_SPEED atau 1/SCALING_SPEED

trus jangan dijadiin parameter, rotate itu tugas paint, bukan control

**Comment by pprogrammer93** [comment]
gimana yang sekarang?

**Comment by yonasadiel** [comment]
kayanya bikin method scaleUp sama scaleDown aja deh, jangan ada parameter, aneh kayanya

**Comment by yonasadiel** [comment]
sekalian nambah dong, jangan rotate left right, tapi clockwise counter clockwise.

hehehe banyak maunya ya

**Comment by pprogrammer93** [comment]
tapi, menurutku rotate left right maknanya sama dan lebih pendek untuk jadi nama fungsi

**Pull request merged**